ext/pcre/php_pcre.c

```diff
@@ -722,6 +723,7 @@ PHPAPI pcre_cache_entry* pcre_get_compiled_regex_cache_ex(zend_string *regex, bo
 			/* PCRE specific options */
 			case 'A':	coptions |= PCRE2_ANCHORED;		break;
 			case 'D':	coptions |= PCRE2_DOLLAR_ENDONLY;break;
+			case 'r':	eoptions |= PCRE2_EXTRA_CASELESS_RESTRICT; break;
```

Comment:
might need to be done conditionally, break with `--with-external-pcre` with older pcre2 version.

Reply:
Good catch; do you prepare a PR or do I?

Reply:
feel free :)

Reply:
@nielsdos did this in #13662, thank you!